utils/prezly/api/queries.ts

```diff
@@ -10,10 +10,21 @@ export const getSlugQuery = (slug: string) => ({
     ],
 });
 
-export const getStoriesQuery = () => ({
-    $and: [
-        ...publishedAndPublic,
-    ],
-});
+export const getStoriesQuery = (newsroomId: number, categoryId?: number) => {
+    const query: any = {
+        $and: [
+            ...publishedAndPublic,
+            { 'newsroom.id': { $in: [newsroomId] } },
+        ],
+    };
+
+    if (categoryId) {
+        query.$and.push(
+            // TODO: filter by category
```

Comment:
@digitalbase @yuriyyakym I'm not an expert on this json queries, do you guys know how to filter by category id here? I can't use the same as line 17 because this should search for an ID in a categories array (`storie.categories: [c1, c2...]`)

Reply:
@e1himself can probably help with this too

Reply:
I added a manual filter while we figure these filters out

-> https://github.com/prezly/theme-nextjs-starter/pull/7/commits/4e7dac0b2410624f49ac598b75be338bb6ce81bb

Reply:
@pablopunk please check the documentation available here: 
https://developers.prezly.com/docs/api/reference/api/API.v2.yaml/paths/~1stories~1search/post

Reply:
thanks @e1himself ! I didn't even know about these docs ❤️

Reply:
@pablopunk From now on please use the documentation portal as your primary source of truth for interactive with the APIs. If there's something missing or incorrect, please file a bug. That's the only way we can keep it up-to-date :)